ocp_resources/machine_config_pool.py

```diff
@@ -2,8 +2,72 @@
 
 
 class MachineConfigPool(Resource):
+    """
+    MachineConfigPool object. API reference:
+    https://docs.openshift.com/container-platform/4.12/rest_api/machine_apis/machineconfigpool-machineconfiguration-openshift-io-v1.html
+
+    Args:
```

Comment:
Let's make is simple, please remove all things from docstring leave only what your resource needs.
in `__init__` get only what your resource needs and then **kwargs

Reply:
Ack.
I think I got it now.